src/FS.Physical/PhysicalFileProvider.cs

```diff
@@ -101,37 +100,7 @@ public bool UsePollingFileWatcher
             set => _usePollingFileWatcher = value;
         }
 
-        /// <summary>
-        /// Gets or sets a value that determines if this instance of <see cref="PhysicalFileProvider"/>
-        /// actively polls for file changes.
-        /// <para>
-        /// When <see langword="true"/>, <see cref="IChangeToken"/> returned by <see cref="Watch(string)"/> will actively poll for file changes
-        /// (<see cref="IChangeToken.ActiveChangeCallbacks"/> will be <see langword="true"/>) instead of being passive.
-        /// </para>
-        /// <para>
-        /// This property is only effective when <see cref="UsePollingFileWatcher"/> is set.
-        /// </para>
-        /// </summary>
-        /// <value>
-        /// The default value of this property is determined by the value of environment variable named <c>DOTNET_USE_POLLING_FILE_WATCHER</c>.
-        /// When <c>true</c> or <c>1</c>, this property defaults to <c>true</c>; otherwise false.
-        /// </value>
-        public bool UseActivePolling
```

Comment:
I took these options out - we'll hopefully have ample time to sort bugs with the polling and Timer our before RTM. Supporting multiple modes just makes the code harder to work with.